pkg/component/etcd/service/values.go

```diff
@@ -28,14 +30,14 @@ type Values struct {
 	ClientServiceAnnotations map[string]string
 	// ClientServiceLabels are the labels to be added to the client service
 	ClientServiceLabels map[string]string
-	// EtcdName is the name of the etcd resource.
-	EtcdName string
-	// EtcdName is the UID of the etcd resource.
-	EtcdUID types.UID
 	// Labels are the service labels.
 	Labels map[string]string
 	// PeerServiceName is the name of the service responsible for peer traffic.
 	PeerServiceName string
 	// ServerPort is the port used for etcd peer communication.
 	ServerPort int32
```

Comment:
Can you rename this to `PeerPort`?

Reply:
Fixed - https://github.com/gardener/etcd-druid/pull/559/commits/b3c08685f1e9cf2ff62917bca8c3e77d69125f97